### PR TITLE
doc: Removed references to Node.js lower than 10.x.

### PR DESCRIPTION
**Pull request by NickNaso**
Find places were we document which versions of Node.js are support and remove all lower than
10.x.
Refs: https://github.com/nodejs/node-addon-api/issues/689

### CODE REVIEW

**Comment by mhdawson** [comment]
What you had looked good but I think we need to say somewhere specifically what is/is not supported.  I pushed a commit which contains my suggestion.

File: README.md [hunk 1]
```diff
@@ -47,14 +47,21 @@ to ideas specified in the **ECMA262 Language Specification**.
...
-## **Current version: 2.0.0**
+## **Current version: 3.0.0**
 
 (See [CHANGELOG.md](CHANGELOG.md) for complete Changelog)
 
 [![NPM](https://nodei.co/npm/node-addon-api.png?downloads=true&downloadRank=true)](https://nodei.co/npm/node-addon-api/) [![NPM](https://nodei.co/npm-dl/node-addon-api.png?months=6&height=1)](https://nodei.co/npm/node-addon-api/)
 
 <a name="setup"></a>
 
+node-addon-api is based on the https://nodejs.org/api/n-api.html and supports using different N-API versison.
```
Comment (mhdawson) [inline README.md:58]:
```
node-addon-api is based on the https://nodejs.org/api/n-api.html and supports using different N-API versions. 
```

File: README.md [hunk 2]
```diff
@@ -47,14 +47,21 @@ to ideas specified in the **ECMA262 Language Specification**.
...
-## **Current version: 2.0.0**
+## **Current version: 3.0.0**
 
 (See [CHANGELOG.md](CHANGELOG.md) for complete Changelog)
 
 [![NPM](https://nodei.co/npm/node-addon-api.png?downloads=true&downloadRank=true)](https://nodei.co/npm/node-addon-api/) [![NPM](https://nodei.co/npm-dl/node-addon-api.png?months=6&height=1)](https://nodei.co/npm/node-addon-api/)
 
 <a name="setup"></a>
 
+node-addon-api is based on the https://nodejs.org/api/n-api.html and supports using different N-API versions.
```
Comment (mhdawson) [inline README.md:58]:
```suggestion
node-addon-api is based on [N-API](https://nodejs.org/api/n-api.html) and supports using different N-API versions. 
```

File: README.md [hunk 3]
```diff
@@ -47,14 +47,21 @@ to ideas specified in the **ECMA262 Language Specification**.
...
 
 (See [CHANGELOG.md](CHANGELOG.md) for complete Changelog)
 
 [![NPM](https://nodei.co/npm/node-addon-api.png?downloads=true&downloadRank=true)](https://nodei.co/npm/node-addon-api/) [![NPM](https://nodei.co/npm-dl/node-addon-api.png?months=6&height=1)](https://nodei.co/npm/node-addon-api/)
 
 <a name="setup"></a>
 
+node-addon-api is based on [N-API](https://nodejs.org/api/n-api.html) and supports using different N-API versions. 
+This allows addons built with it to run with Node.js versions which support the targetted N-API version. 
+**However** node-addon-api the support model is to support only the active LTS Node.js versions. This means that
```
Comment (mhdawson) [inline README.md:60]:
```
**However** the node-addon-api support model is to support only the active LTS Node.js versions. This means that
```

File: README.md [hunk 4]
```diff
@@ -47,14 +47,21 @@ to ideas specified in the **ECMA262 Language Specification**.
...
 [![NPM](https://nodei.co/npm/node-addon-api.png?downloads=true&downloadRank=true)](https://nodei.co/npm/node-addon-api/) [![NPM](https://nodei.co/npm-dl/node-addon-api.png?months=6&height=1)](https://nodei.co/npm/node-addon-api/)
 
 <a name="setup"></a>
 
+node-addon-api is based on [N-API](https://nodejs.org/api/n-api.html) and supports using different N-API versions. 
+This allows addons built with it to run with Node.js versions which support the targetted N-API version. 
+**However** the node-addon-api support model is to support only the active LTS Node.js versions. This means that
+every year there will be a new major which drops support for the Node.js LTS version which has gone out of service.
+
+The oldest Node.js version that the current node-addon-version supports is currently Node.js 10.x.
```
Comment (mhdawson) [inline README.md:63]:
```suggestion
The oldest Node.js version supported by the current version of node-addon-api is Node.js 10.x.
```

File: README.md [hunk 5]
```diff
@@ -47,14 +47,21 @@ to ideas specified in the **ECMA262 Language Specification**.
...
+## **Current version: 3.0.0**
 
 (See [CHANGELOG.md](CHANGELOG.md) for complete Changelog)
 
 [![NPM](https://nodei.co/npm/node-addon-api.png?downloads=true&downloadRank=true)](https://nodei.co/npm/node-addon-api/) [![NPM](https://nodei.co/npm-dl/node-addon-api.png?months=6&height=1)](https://nodei.co/npm/node-addon-api/)
 
 <a name="setup"></a>
 
+node-addon-api is based on [N-API](https://nodejs.org/api/n-api.html) and supports using different N-API versions. 
+This allows addons built with it to run with Node.js versions which support the targetted N-API version.
```
Comment (kgryte) [inline README.md:59]:
```suggestion
This allows addons built with it to run with Node.js versions which support the targeted N-API version. 
```

**Comment by mhdawson** [comment]
@NickNaso are you ok with the additions I made? If so I'll go ahead and land it.

**Comment by NickNaso** [comment]
> @NickNaso are you ok with the additions I made? If so I'll go ahead and land it.

@mhdawson Yes, I'm ok. Please land it.

**Pull request merged**